newIDE/app/src/ObjectEditor/Editors/SpriteEditor/PointsEditor/PointRow.js

```diff
@@ -32,114 +31,110 @@ type Props = {|
-                id="point-x"
-                onChange={value => {
-                  const valueAsNumber = parseFloat(value);
-                  if (!isNaN(valueAsNumber))
-                    props.onChangePointX(valueAsNumber);
-                }}
-                onBlur={event => {
-                  props.onChangePointX(
-                    parseFloat(event.currentTarget.value) || 0
-                  );
-                }}
-              />
-            ) : (
-              <Text noMargin>
-                <Trans>(auto)</Trans>
-              </Text>
-            )}
-          </Column>
-        </TableRowColumn>
-        <TableRowColumn style={styles.coordinateColumn} padding="none">
-          <Column>
-            {!props.isAutomatic ? (
-              <SemiControlledTextField
-                margin="none"
-                inputStyle={
-                  props.selected
-                    ? { color: muiTheme.listItem.selectedTextColor }
-                    : undefined
-                }
-                value={roundTo(pointY, POINT_COORDINATE_PRECISION).toString()}
-                type="number"
-                id="point-y"
-                onChange={value => {
-                  const valueAsNumber = parseFloat(value);
-                  if (!isNaN(valueAsNumber))
-                    props.onChangePointY(valueAsNumber);
-                }}
-                onBlur={event => {
-                  props.onChangePointY(
-                    parseFloat(event.currentTarget.value) || 0
-                  );
-                }}
-              />
-            ) : (
-              <Text noMargin>
-                <Trans>(auto)</Trans>
-              </Text>
-            )}
-          </Column>
-        </TableRowColumn>
-        <TableRowColumn style={styles.toolColumn}>
-          {!!props.onRemove && (
-            <IconButton size="small" onClick={props.onRemove}>
-              <Delete />
-            </IconButton>
+const PointRow = ({ pointX, pointY, ...props }: Props) => {
+  const muiTheme = React.useContext(GDevelopThemeContext);
+  return (
+    <TableRow
+      style={{
+        backgroundColor: props.selected
+          ? muiTheme.listItem.selectedBackgroundColor
+          : muiTheme.list.itemsBackgroundColor,
+      }}
+      onClick={() => props.onClick(props.pointName)}
+      onPointerEnter={() => props.onPointerEnter(props.pointName)}
+      onPointerLeave={props.onPointerEnter}
+    >
+      <TableRowColumn style={styles.nameColumn}>
+        <SemiControlledTextField
+          margin="none"
+          inputStyle={
+            props.selected
+              ? { color: muiTheme.listItem.selectedTextColor }
+              : undefined
+          }
+          value={props.pointName}
+          id={props.pointName}
+          fullWidth
+          errorText={props.nameError ? 'This name is already taken' : undefined}
+          disabled={!props.onBlur}
+          onBlur={props.onBlur}
+          onChange={value => {}}
```

Comment:
This can be replaced by:

```suggestion
          disabled={!props.onChangePointName}
          commitOnBlur
          onChange={props.onChangePointName}
```

if you replace `onBlur?: (ev: any) => void,` by `onChangePointName: (newName: string) => void` and adapt the caller (there is only one place where to change this).